src/components/Column.test.tsx

```diff
@@ -67,3 +74,101 @@ test("Change Column width", async () => {
   userEvent.click(await screen.findByLabelText("decrease width"));
   expect(screen.queryByLabelText("decrease width")).toBeNull();
 });
+
+test("Show Referenced By", async () => {
+  const [alice] = setup([ALICE]);
+  const aliceKnowledgeDB = await setupTestDB(alice(), [["Money", ["Bitcoin"]]]);
+  const btc = findNodeByText(aliceKnowledgeDB, "Bitcoin") as KnowNode;
+  const db = await setupTestDB(alice(), [
+    ["Alice Workspace", [[btc], ["P2P Apps", [btc]]]],
+  ]);
```

Comment:
i guess that could be one line with one setupTestDB, no?

Reply:
no, we cannot do that. As I need "Bitcoin" three times, I first need to create a reference to it, If I would do [["bitcoin"], ["P2P Apps", ["Bitcoin"]], setup would just create new "Bitcoin" nodes